src/components/PayeeList.tsx

```diff
@@ -0,0 +1,41 @@
+import { HorizontalLabelledField } from '@components'
+import { Payee } from '@types'
+import * as React from 'react'
+
+type RemovePayee = (payeeName: string) => void
+export interface Props {
+  payees: Payee[]
+  removePayee: RemovePayee
+}
+
+export const deletePayeeLink: React.SFC<{name: string, removePayee: RemovePayee}> = ({name, removePayee}) => {
+  return (
+    <a
+      className='delete'
+      style={{marginTop: '3.5px'}}
+      onClick={() => removePayee(name)}
+    />
+  )
+}
+
+export const PayeeList: React.SFC<Props> = (props) => {
+  const payees = props.payees.map((payee) => {
+    return (
+      <HorizontalLabelledField
+        label={payee.name}
+        key={payee.name}
+        value={payee.publicKey}
+        addon={deletePayeeLink({name: payee.name, removePayee: props.removePayee})}
```

Comment:
Once it is capitalised you would call it like `<DeletePayee removePayee={() => props.removePayee(payee.name)} />`